coldfront/core/user/templates/account_deactivation/request_list.html

```diff
@@ -0,0 +1,101 @@
+{% load common_tags %}
+{% load static %}
+
+
+{% block title %}
+  Cluster Account Deactivation Requests
+{% endblock %}
+
+
+{% block content %}
+  <script type='text/javascript' src="{% static 'selectize/selectize.min.js' %}"></script>
+  <link rel='stylesheet' type='text/css' href="{% static 'selectize/selectize.bootstrap3.css' %}"/>
+
+  <h1>{{ status|title }} Cluster Account Deactivation Requests</h1>
+
+  <div class="table-responsive">
+    <table class="table table-sm">
+      <tbody>
+      <tr>
+        <td>
+          Viewing {{ status|lower }} cluster account deactivation requests. To view other requests, use
+          the search function below.
+        </td>
+        <td>
+        </td>
+      </tr>
+      </tbody>
+    </table>
+  </div>
+
+  {% if expand_accordion %}
+    <div class="mb-3" id="accordion">
+      <div class="card">
+        <div class="card-header">
+          <a id="expand_button" role="button" class="card-link " data-toggle="collapse" href="#collapseOne">
+            <i class="fas fa-filter" aria-hidden="true"></i> Search
+            <i id="plus_minus" class="fas {{expand_accordion|get_icon}} float-right"></i>
+          </a>
+        </div>
+        <div id="collapseOne" class="collapse {{expand_accordion}}" data-parent="#accordion">
+          <div class="card-body">
+            <form id="filter_form" method="GET" action="{% url 'account-deactivation-request-list' %}" autocomplete="off">
+              {{ search_form|crispy }}
+              <input type="submit" class="btn btn-primary" value="Search">
+              <button id="form_reset_button" type="button" class="btn btn-secondary">Reset</button>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+  {% endif %}
+
+  {% if account_deactivation_requests %}
+    <div class="table-responsive">
+      {% include 'account_deactivation/reason_card.html' %}
+      <p></p>
+      {% with queryset=account_deactivation_requests %}
+        {% include 'account_deactivation/request_list_table.html' %}
+      {% endwith %}
+      {% if is_paginated %}
+        {% include 'common/pagination.html' %}
+      {% endif %}
+    </div>
+  {% else %}
+    <div class="alert alert-info">
+        No {{ status|lower }} cluster account deactivation requests!
+    </div>
+  {% endif %}
+
+  <script>
+      $("#navbar-main > ul > li.active").removeClass("active");
+      $("#navbar-admin").addClass("active");
+      $("#navbar-allocation-cluster-account-deactivation-requests").addClass("active");
+
+      $('select').selectize({
+          create: false
+      })
+
+      $(document).on('click', '#form_reset_button', function() {
+          resetForm($('#filter_form'));
+      });
+
+      function resetForm($form) {
+          $form.find('input:text, input:password, input:file, select, textarea').val('');
+          $form.find("input[type='number']").val('');
+
+          for (let i = 0; i < 6; i++) {$('select')[i].selectize.clear();}
```

Comment:
Magic number.

You should be able to use jQuery to count the number of `select` elements, or just iterate over all of them directly.